queries/rbs/indents.scm

```diff
@@ -0,0 +1,4 @@
+[(class) (module) (interface)] @indent.begin
```

Comment:
Class, modules, interfaces, so there should be methods as well? And probably also branches, and such